developers/contracts.mdx

```diff
@@ -6,22 +6,64 @@ For mev-commit testnet contract addresses please refer to [Testnet](/developers/
 
 ## Core Contract Architecture and Functionality
 
-The mev-commit protocol uses 5 core contracts:
-
-- **Preconfirmations Contract**
-  - This will persist details about the commitments that have been made into contract storage.
-- **Bidder Registry Contract**
-  - This will be where the bulk of monetary transfer occurs.
-  - Bidders deposit for bids into this contract.
-  - The payment gets processed into a provider-allocated section that a provider can later retrieve.
-- **Provider Registry Contract**
-  - This contract stores the stake a provider bounds to ensure it doesn't break its promises under preconfirmations.
-- **Oracle Contract**
-  - This simply retrieves details on which commitments to process for rewarding or slashing.
-  - Access to the functions on this contract is restricted to the owner of the Oracle key.
-- **Block Tracker Contract**
-  - This will track Ethereum blocks and their winners, so commitments could be rewarded or slashed
-  - Access to the functions on this contract is restricted to the owner of the Oracle key.
+The mev-commit protocol uses 6 core contracts. Only the validator contract is on L1, the other contracts are on `mev-commit` chain:
```

Comment:
There are only 5 of what we have termed "core contracts", all of which are deployed on the mev-commit chain https://github.com/primev/mev-commit/tree/main/contracts/contracts/core